packages/store/src/patterns/encodeKey.js

```diff
@@ -85,17 +85,37 @@ const dbEntryKeyToNumber = k => {
   return result;
 };
 
-// BigInts are encoded as keys as follows:
-//   `${prefix}${length}:${encodedNumber}`
+// Positive BigInts are encoded as keys as follows:
+//   `p${lenlenm1}:${length}:${encodedNumber}`
 // Where:
 //
-//   ${prefix} is either 'n' or 'p' according to whether the BigInt is negative
-//      or positive ('n' is less than 'p', so negative BigInts will sort below
-//      positive ones)
+//   Where the 'p' indicates "positive". Below we explain the current encoding
+//      of negative bigints, which begin with 'n'. 'n' is less than 'p',
+//      so negative BigInts will sort below positive ones.
 //
 //   ${encodedNumber} is the value of the BigInt itself, encoded as a decimal
 //      number.  Positive BigInts use their normal decimal representation (i.e.,
-//      what is returned when you call `toString()` on a BigInt).  Negative
+//      what is returned when you call `toString()` on a BigInt).
+//
+//   ${length} is the decimal representation of the width (i.e., the count of
+//      digits) of the BigInt.
+//
+//   ${lenlenm1} is the length of the length minus 1, which must
+//      only be a single digit. (We subtract 1 because the length of the
+//      length will never be zero.)
+//      This limits us to bigints smaller that 10 ** 10 ** 10, which is
```

Comment:
I maintain that it's smaller than `11 ** 10 ** 10` ;)

Reply:
`10 ** 10 ** 11` ;)

Done.